ci-operator/step-registry/hypershift-qe/aws/hypershift-qe-aws-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: hypershift-qe-aws
+  from: tests-private
```

Comment:
`hypershift` cli is only available in `tests-private` image?

Reply:
Yes, there is no official release of HyperShift client. `tests-private` is an `openshift-test-private` repo image where HyperShift agent is installed inside.